src/bn256/mod.rs

```diff
@@ -36,6 +42,29 @@ use error::Error;
 pub struct Bn256;
 
 impl Bn256 {
+    /// Function to calculate the modulus of a U256.
+    ///
+    /// # Arguments
+    ///
+    /// * `num` - the number we want to reduce.
+    /// * `modulus` - the modulus we want to apply.
+    ///
+    /// # Returns
+    ///
+    /// * If successful, a `U256` representing num % modulus.
+
```

Comment:
Remove this empty line. And since this function does not need self, you can move it outside of `impl Bn256` and remove `&self`.